Vector.cpp

```diff
@@ -1,8 +1,21 @@
 #include "Vector.h"
 #include <assert.h>
+#include <iostream>
+
+using namespace std;
 
 namespace Micromouse
 {
+	direction operator%(const direction &dir1, const direction &dir2)
```

Comment:
That works exactly as I intended. It's really easy for me to visualize, but difficult to write out. Suppose I'm facing direction X and I want to turn 45 degrees to the right. I would use X = X % NE, since NE is 45 degrees to the right of north. If I wanted to turn around, I would use X = X % S since south is in the opposite direction of north.